### ISSUE TITLE
Error occurred, while trying to send channel message

### ISSUE THREAD

**Issue by katastrophe411** [microsoftgraph/msgraph-sdk-java#390]
Hi!
I am using MS Graph java sdk v1.8
Trying to send channel message as follows
```
ChatMessage newChatMessage = new ChatMessage();
        ItemBody body = new ItemBody();
        body.contentType = BodyType.TEXT;
        body.content = String.format("Hi!");
        newChatMessage.body = body;
graphClient.teams('{id}")
         .channels("{id}")
         .messages()
         .buildRequest()
         .post(newChatMessage);
```

I get error as follows
```
java.lang.IllegalArgumentException: class com.microsoft.graph.models.extensions.ChatMessageMention declares multiple JSON fields named id
	at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory.getBoundFields(ReflectiveTypeAdapterFactory.java:170)
	at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory.create(ReflectiveTypeAdapterFactory.java:100)
	at com.google.gson.Gson.getAdapter(Gson.java:423)
	at com.google.gson.internal.bind.CollectionTypeAdapterFactory.create(CollectionTypeAdapterFactory.java:53)
	at com.google.gson.Gson.getAdapter(Gson.java:423)
	at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory.createBoundField(ReflectiveTypeAdapterFactory.java:115)
	at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory.getBoundFields(ReflectiveTypeAdapterFactory.java:164)
	at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory.create(ReflectiveTypeAdapterFactory.java:100)
```

ChatMessageMention.class
```
public class ChatMessageMention extends Entity implements IJsonBackedObject {
    /**
     * The Id.
     * 
     */
    @SerializedName("id")
    @Expose
    public Integer id;
....
```

Entity.class
```
public class Entity implements IJsonBackedObject {
...
    /**
     * The Id.
     * Read-only.
     */
    @SerializedName("id")
    @Expose
    public String id;
```

**Comment by Shubh-arya** [comment]
Hi There,

I am facing the same issue, could you please let me know if there is any work around available for the same.

Thanks,
Shubham

**Comment by mfbieber** [comment]
Hi,

I see the same error while trying to deserialize Channel Messages.  The model classes are generated from the metadata retrieved from MsGraph, correct? It has changed from (https://raw.githubusercontent.com/microsoftgraph/MSGraph-SDK-Code-Generator/dev/metadata/2018_10_23/beta_2018_10_23_source.xml)
```
<ComplexType Name="chatMessageMention">
    <Property Name="type" Type="microsoft.graph.chatMessageMentionType" Nullable="false" />
    <Property Name="id" Type="Edm.String" />
    <Property Name="mentionText" Type="Edm.String" />
</ComplexType>
```

to (retrieved today from https://graph.microsoft.com/v1.0/$metadata):
```
<ComplexType Name="chatMessageMention">
    <Property Name="id" Type="Edm.Int32" />
    <Property Name="mentionText" Type="Edm.String" />
    <Property Name="mentioned" Type="graph.identitySet" />
</ComplexType>
```

Did it work before when the id had the Type `Edm.String`?

**Comment by Shubh-arya** [comment]
What is timeline for 1.10 release? If there is a pre-defined release date let me know.

**Comment by baywet** [comment]
Hey folks!
Thanks for your patience on that matter.
I can confirm this was fixed by #402 and that it now works with version 2.0.0.
Closing
